### PR TITLE
Moved username and key to capabilities + tests

### PR DESCRIPTION
**Pull request by BeyondEvil**
Moved Sauce specific tests from `test_capabilities.py` to `test_saucelabs.py` as well.

### CODE REVIEW

File: testing/test_saucelabs.py [hunk 1]
```diff
@@ -77,6 +78,83 @@ def test_invalid_credentials_file(failure, monkeypatch, tmpdir):
...
+    capabilities = {'browserName': 'chrome'}
+    variables = testdir.makefile('.json', '{{"capabilities": {}}}'.format(
+        json.dumps(capabilities)))
+
+    file_test = testdir.makepyfile("""
+        import pytest
+        @pytest.mark.nondestructive
+        def test_sauce_capabilities(driver_kwargs):
+            assert driver_kwargs['desired_capabilities']['username'] == '{}'
+            assert driver_kwargs['desired_capabilities']['accessKey'] == '{}'
```
Comment (davehunt) [inline testing/test_saucelabs.py]: The purpose of this test is to check that `sauce:options` is not present, so I don't think we need to check the capabilities here.

**Comment by davehunt** [comment]
@BeyondEvil I'm happy for you to merge this with or without my comments addressed. Thanks!

File: testing/test_saucelabs.py (hunk 1 above)
Comment (BeyondEvil) [reply testing/test_saucelabs.py]: Dammit, I should've followed my instincts. Will move them out to their own test. 👍 

Will add a section to the release-notes as well. :)

File: docs/news.rst [hunk 2]
```diff
@@ -1,6 +1,14 @@
 Release Notes
 =============
 
+
+TBD
```
Comment (davehunt) [inline docs/news.rst]: We can assume it will be 1.14.0 and I typically just set the date as "(unreleased)".

**Pull request merged**